client/src/components/ActivityShow.js

```diff
@@ -27,7 +28,7 @@ const ActivityShow = (props) => {
 
   useEffect(() => {
     getActivity();
-  }, []);
+  }, [reviewList]);
```

Comment:
this should not be required - when a new review is added to the `reviewList` we do not need to refetch all the data about the activity and it's related reviews - that information is already in state, and after the POST request we add the new review to state